web/pgadmin/preferences/static/js/components/PreferencesComponent.jsx

```diff
@@ -292,6 +292,16 @@ export default function PreferencesComponent({ ...props }) {
           opt.selected = false;
         }
       });
+
+      let sync_with_os_id = null;
+      subNode.preferences.forEach((_el) => {
```

Comment:
you can remove this once you the same `theme` preference.

Reply:
Done.